06-building-a-program-start-to-finish/writing-tests/output/inverted-index.spec.js

```diff
@@ -0,0 +1,98 @@
+const InvertedIndex = require('../js/inverted-index');
+const model = require('./testmodels.js');
+
+describe('Test suite for Inverted Index Class', () => {
+  
+  const InvertedIndexTest = new InvertedIndex();
+  
+  describe('Should test for the instance of InvertedIndexTest', () => {
```

Comment:
Your test suites are not named properly.

I should be able to read your suites and your specs and make somewhat of a proper sentence out of it.
```
'Should test for the instance of InvertedIndexTest'
  Should return true if InvertedIndexTest is an instance of Object
```
The above doesn't exactly make a sentence. Feel free to research more on Behaviour-Driven Development.